src/main/java/vbn/state/State.java

```diff
@@ -110,6 +110,10 @@ public void pushConstraint(UnaryOperand operand, String symbol) {
         pushConstraint(constraint);
     }
 
+    public void pushConstraints(Stack<Constraint> constraints) {
```

Comment:
This looks a little sketchy. Could you double-check?

Reply:
Fixed in [2f8565f](https://github.com/CMPT-479/CMPT-479-Project/pull/32/commits/2f8565f4f3df431c55656d09b9ba55c80fefca89)